Formula/xplanet.rb

```diff
@@ -37,6 +37,11 @@ class Xplanet < Formula
   depends_on "freetype"
   depends_on :x11 => :optional
 
+  patch :p2 do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/f952f1d/xplanet/xplanet-1.3.1-ntimes.patch"
```

Comment:
@blogabe Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

@woodruffw Just FYI for future that that's a blocker on accepting patches.